3001-4000/LIT3177Canon.xml

```diff
@@ -25,7 +28,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </sourceDesc>
          <sourceDesc>
             <listWit>
-               <witness/>
+              <witness corresp="EMIP02264#ms_i3"/>
```

Comment:
This encoding of witnesses does not correspond (anymore) to the guidelines: only in case of an edition present in this file should the witnesses used for this edition be encoded here (https://betamasaheft.eu/Guidelines/?id=work-teiHeader).
If these manuscripts will not be described anythime soon and the work ID therefore not be used in their description and you want to keep the information that they contain this work, you could add the relation lawd:hasAttestation.

Reply:
I think that I have done this correctly - please let me know if there are any further errors.